content/en/docs/project-user-guide/storage/volumes.md

```diff
@@ -1,10 +1,46 @@
 ---
 title: "Volumes"
-keywords: 'kubernetes, docker, helm, jenkins, istio, prometheus'
+keywords: 'kubernetes, docker, persistent volume, persistent volume claim, volume clone, volume snapshot, volume expanding'
 description: 'Create Volumes (PVCs)'
 
 linkTitle: "Volumes"
 weight: 2110
 ---
 
-TBD
+## Introduction
+In this section, volumes always refer to PersistentVolumeClaim(PVC) of Kubernetes.
+
+## Create Volume
+### Method
+There are two methods to create volume: 
+- Create empty volume by StorageClass
+- Create volume from VolumeSnapshot 
+
+![Create](/images/storage/create-volume.png)
+
+## Attach Volume onto Workloads
+Take attaching volume onto deployment for example, in the `Mount Volume` step of *Create Deployment*, 
+volumes cloud be attached on containers' path.
+![Attach](/images/storage/attach-volume.png)
+
+## Volume Features
+Volume Features include: 
+- Clone Volume
+- Create Volume Snapshot
+- Expand Volume 
+
+KubeSphere can get supported features of underlying storage plugin that called `Storage Capability`.
+The console display only supported features in `Volume Detail Page`.
+More about `Storage Capability` see [design](https://github.com/kubesphere/community/blob/master/sig-storage/concepts-and-designs/storage-capability-interface.md)
```

Comment:
More about `Storage Capability` see [design](https://github.com/kubesphere/community/blob/master/sig-storage/concepts-and-designs/storage-capability-interface.md)
->
For more information about `Storage Capability`, see [Design](https://github.com/kubesphere/community/blob/master/sig-storage/concepts-and-designs/storage-capability-interface.md#Design).